service-python/assessclaimdc7101/src/lib/conditions.py

```diff
@@ -21,9 +21,13 @@ def conditions_calculation(request_body):
     for condition in veterans_conditions:
         condition_code = condition["code"]
         if "recordedDate" in condition.keys():
-            date = datetime.strptime(condition["recordedDate"], "%Y-%m-%d").date()
-            condition["dateFormatted"] = date.strftime("%m/%d/%Y")
-            condition_with_date.append(condition)
+            if condition["recordedDate"]:
+                date = datetime.strptime(condition["recordedDate"], "%Y-%m-%d").date()
```

Comment:
what should be the expected behavior if the date cannot be parsed? As it is now, it will throw an uncaught exception

Reply:
I will add a try and except